docs/sources/setup-grafana/set-up-https.md

```diff
@@ -0,0 +1,278 @@
+---
+aliases:
```

Comment:
I think we should remove this key from the front matter since we don't have any aliases.

We need to remove this line and line three.

Reply:
great, thx!